app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -49,6 +50,10 @@ import org.mozilla.fenix.utils.Settings
 @SuppressLint("Registered")
 @Suppress("TooManyFunctions", "LargeClass")
 open class FenixApplication : LocaleAwareApplication() {
+    init {
+        recordOnInit() // DO NOT MOVE ANYTHING ABOVE HERE: the timing of this measurement is critical.
+    }
```

Comment:
It's somewhat scary to run all of that code without any crash reporting. There's probably not much we can do. But it may be another reason to shorten the lifetime of this measurement.

Reply:
That's a very good point I did not consider. Luckily this is a few assignments and an if statement and I don't expect the code to change as much. I agree with shortening the lifetime though.

Reply:
This is covered by Google Play crash services though, right? Perhaps I should monitor that when this lands.